components/Dialogs/src/Theming/MDCDialogPresentationController+MaterialTheming.m

```diff
@@ -27,8 +27,15 @@ - (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
   }
   self.scrimColor = [colorScheme.onSurfaceColor colorWithAlphaComponent:(CGFloat)0.32];
 
-  // Other properties
-  self.dialogCornerRadius = kCornerRadius;
+  // Corner Radius
+  if ([self.presentedViewController isKindOfClass:[MDCAlertController class]]) {
+    ((MDCAlertController *)self.presentedViewController).cornerRadius = kCornerRadius;
+  } else {
```

Comment:
My concern with this approach is that by accessing the view property of the presentedViewController we may trigger "viewDidLoad" inside the themer.  
Some solutions are:
*  I could call isViewLoaded first, but then the corner radius will have to be set elsewhere after the view is loaded.
* set the corner radius in the presentation controller's presentationTransitionWillBegin which is called when the view is being presented (should we match the view.layer.cornerRadius to be the same as dialogCornerRadius, or the other way around?)
* Just document this fact in the theming extension's header file.  

Any suggestions/thoughts?